src/python/grpcio/grpc/_common.py

```diff
@@ -97,3 +98,27 @@ def serialize(message, serializer):
 def deserialize(serialized_message, deserializer):
   return _transform(serialized_message, deserializer,
                     'Exception deserializing message!')
+
+
+class CleanupThread(threading.Thread):
+  """A wrapper around thread for custom join behavior.
+  
+  On Interpreter exit, Python will attempt to join outstanding threads
```

Comment:
"Interpreter" should only get a capital i when the word "Python" immediately precedes it.